styleguide/source/_patterns/01-molecules/12-ec/02-ec-bundle-card/_ec-bundle-card.scss

```diff
@@ -0,0 +1,101 @@
+.ec-bundle-card {
+  display: flex;
+  flex-direction: column;
+  background-color: $black-7;
+  color: $black;
+  padding: 24px;
+  width: 100%;
+  margin-bottom: 20px;
+  text-align: center;
+
+  @include type($font-sans-serif, 18px, $font-weight-medium, 1.3888888889);
+
+  @include breakpoint($bp-med) {
+    @include type($font-sans-serif, 20px, $font-weight-medium, 1.3333333333);
+  }
+
+  &:hover {
+    background-color: $gray-light-4;
+    color: $black;
+  }
+
+  @include breakpoint($bp-small $bp-med) {
+    flex-direction: row;
+
+    .ec-bundle-card_image {
+      display: inline-block;
+      max-width: 100%;
+      height: auto;
+      @include grid__unit--cols(4);
+    }
+
+    .ec-bundle-card_text {
+      display: inline-block;
+      margin: 0 24px;
+      text-align: left;
+      @include grid__unit--cols(8);
+    }
+  }
+
+  @include breakpoint($bp-med) {
+    // 2-up layout
+    // when there are three items it's 50:50
```

Comment:
Card patterns should only include styling for the card itself, and then larger patterns that include card patterns should introduce the row CSS. We can talk more about this when we discuss reorganizing the cards, but for now, having this layout/card row CSS in this pattern might actually make my work to implement the cards in the card rows more complicated :| 

If it isn't too much work, can you remove it for now? Again, if you disagree with this, I'm open to discussion when we do that spike.